lib/binance/util.ex

```diff
@@ -0,0 +1,35 @@
+defmodule Binance.Util do
+  @moduledoc false
+
+  @doc """
+  Prepare request body, sign it if required
+  """
+  def prepare_request_body(params, sign?) do
```

Comment:
I would suggest use `opts` for example: 

`prepare_request_body(params, opts \\ [])` => more descriptive. So when I use the function I have to use something like

`Util.prepare_request_body(params, [signed: true])`